include/aspect/stokes_matrix_free.h

```diff
@@ -84,6 +84,18 @@ namespace aspect
                              const double pressure_scaling,
                              const bool is_compressible);
 
+        /**
+         * Fills in the tables of derivatives and strain rate for Newton method,
+         * and gives information regarding stabilization schemes.
```

Comment:
Do you mean `and stores information about the symmetry of the derivatives, which is relevant for the stabilization schemes.`?